tsconfig.json

```diff
@@ -6,15 +6,15 @@
     "noEmitOnError": true,
     "outDir": "build/tsc-out",
     "sourceMap": true,
-    "target": "es2017",
-    "lib": ["es2017","es2019.array", "es2018.promise", "dom", "dom.iterable"],
+    "target": "es2019",
+    "lib": ["es2019", "dom", "dom.iterable"],
```

Comment:
Note: [According to caniuse.com](https://caniuse.com/?feats=mdn-javascript_builtins_regexp_dotall,mdn-javascript_builtins_regexp_lookbehind_assertion,mdn-javascript_builtins_regexp_named_capture_groups,mdn-javascript_builtins_regexp_property_escapes,mdn-javascript_builtins_symbol_asynciterator,mdn-javascript_functions_method_definitions_async_generator_methods,mdn-javascript_grammar_template_literals_template_literal_revision,mdn-javascript_operators_destructuring_rest_in_objects,mdn-javascript_operators_spread_spread_in_destructuring,promise-finally), Firefox 68.0 doesn't support a tiny part of ES2018. I don't think it'll cause problems, but it might be good to keep in mind.